xarray/tests/test_cftime_offsets.py

```diff
@@ -54,11 +55,32 @@ def calendar(request):
         (YearEnd(), 1),
         (QuarterBegin(), 1),
         (QuarterEnd(), 1),
+        (Tick(), 1),
+        (Day(), 1),
+        (Hour(), 1),
+        (Minute(), 1),
+        (Second(), 1),
+        (Millisecond(), 1),
+        (Microsecond(), 1),
         (BaseCFTimeOffset(n=2), 2),
         (YearBegin(n=2), 2),
         (YearEnd(n=2), 2),
         (QuarterBegin(n=2), 2),
         (QuarterEnd(n=2), 2),
+        (Tick(n=2), 2),
+        (Day(n=2), 2),
+        (Hour(n=2), 2),
+        (Minute(n=2), 2),
+        (Second(n=2), 2),
+        (Millisecond(n=2), 2),
+        (Microsecond(n=2), 2),
+        (Tick(n=1.5), 1.5),
+        (Day(n=1.5), 1.5),
+        (Hour(n=1.5), 1.5),
+        (Minute(n=1.5), 1.5),
+        (Second(n=1.5), 1.5),
+        (Millisecond(n=1.5), 1.5),
+        (Microsecond(n=1.5), 1.5),
```

Comment:
Instead of these, I think we should explicitly test using the multiplication operator.  E.g. I think you could augment the existing tests [here](https://github.com/pydata/xarray/blob/9d2c1d8959ccb385664294e6a1372a272cddfbc8/xarray/tests/test_cftime_offsets.py#L385-L409) by introducing a new parameter for the `"multiple"`.  

Also please add tests for the following additional cases:
- That the appropriate error is raised if a non-Tick offset is multiplied by a float.
- That the appropriate error is raised if a Tick offset is multiplied by a float which causes it not to be representable by a microsecond-precision timedelta (e.g. the `0.5 * Microsecond()` example).

These might make sense to test in a separate test function.

Reply:
added separate tests

Reply:
Ah sorry, I forgot `"multiple"` could mean more than one thing in this context.  I meant something like the following:
```python
@pytest.mark.parametrize(("offset", "multiple", "expected"), _MUL_TESTS, ids=_id_func)
def test_mul(offset, multiple, expected):
    assert offset * multiple == expected
```
What you have is solid, but using a setup like this might make adding my additional suggested test a little cleaner.

Reply:
for `Tick` and `int`, I could do _MUL_TESTS = [(Day(), 3, Day(n=3))]`.
for `Tick` and `float`, I could do _MUL_TESTS = [(Day(), .5, Hour(n=6))]`

Reply:
added